src/routes/projects.routes.js

```diff
@@ -539,6 +539,39 @@ router.get(
   projectController.getBuildStatusesForProject
 );
 
+router.get(
+  '/projects/:projectId/logs/:buildNumber',
+  authController.isAuthenticated,
+  projectController.getLogsForBuilldForProject
+);
+
+/**
+ * @api {get} /projects/:projectId/heroku Get heroku app info for project
+ * @apiVersion 1.0.0
+ * @apiName get heroku app info for project
+ * @apiGroup Projects
+ *
+ * @apiHeader Authorization Basic Auth-Token
+ * @apiPermission Authenticated User
+ *
+ * @apiParam {String} projectId the projectId
+ *
+ * @apiSuccess {String} url the heroku app url to the deployed application
+ * @apiSuccess {String} updated_at the last time the app was updated on heroku
+ *
+ * @apiSuccessExample {json} Success-Resonse:
+ *
+  {
+    "url": "https://jack-bkuiket.herokuapp.com/",
+    "updated_at": "2018-01-26T18:57:51Z"
+  }
+ */
+router.get(
+  '/projects/:projectId/heroku',
+  authController.isAuthenticated,
```

Comment:
Should we check if they are a contributor on the project or can everyone view this?